packages/core/src/utils/jwks.ts

```diff
@@ -52,3 +54,57 @@ export const exportJWK = async (key: KeyLike | Uint8Array): Promise<JWK> => {
     kid: calculateKid(jwk),
   };
 };
+
+/**
+ * Migrated from `@logto/cli` and used in `/logto-config/rotate-private-keys` API.
+ * TODO: Need to figure out a better way to share code across cli and core packages.
```

Comment:
nit: better add an issue as well.  We have multiple legacy to-do items lost track.

Reply:
Yes, I have created a ticket last week. Here's the link: https://linear.app/silverhand/issue/LOG-7182/share-code-with-node-dependencies